src/nanovg_gl.h

```diff
@@ -345,12 +346,12 @@ static void glnvg__blendFuncSeparate(GLNVGcontext* gl, const GLNVGblend* blend)
 
 static GLNVGtexture* glnvg__allocTexture(GLNVGcontext* gl)
 {
-	GLNVGtexture* tex = NULL;
-	int i;
+ 	GLNVGtexture* tex = NULL;
+	int i, idx = -1, salt;
```

Comment:
int i, idx = -1, salt = 1;

Reply:
The salt as a local variable in allocTexture means that it isn't preserved across calls. I made it a member of GLNVGcontext as textureSalt.